migrate.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS `vehicle_financing` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `citizenid` varchar(50) DEFAULT NULL,
+    `plate` varchar(15) NOT NULL,
+    `balance` int(11) NOT NULL DEFAULT 0,
+    `paymentamount` int(11) NOT NULL DEFAULT 0,
+    `paymentsleft` int(11) NOT NULL DEFAULT 0,
+    `financetime` int(11) NOT NULL DEFAULT 0,

```

Comment:
Why the default values?

Reply:
That's how it was🤷🏻‍♂️, didn't want to break anything (we never know), but i'll change it to NULL